src/view/prsTreeDataProvider.ts

```diff
@@ -64,6 +64,7 @@ export class PullRequestsTreeDataProvider implements vscode.TreeDataProvider<Tre
 				new CategoryTreeNode(this._prManager, this._telemetry, PRType.RequestReview),
 				new CategoryTreeNode(this._prManager, this._telemetry, PRType.AssignedToMe),
 				new CategoryTreeNode(this._prManager, this._telemetry, PRType.Mine),
+				new CategoryTreeNode(this._prManager, this._telemetry, PRType.ReviewedByMe),
```

Comment:
I think this category should go after `PRType.RequestReview`

Reply:
Agreed.